ci-operator/step-registry/3scale/apimanager/install/3scale-apimanager-install-ref.yaml

```diff
@@ -0,0 +1,31 @@
+ref:
+  as: 3scale-apimanager-install
+  from: 3scale-runner
+  commands: 3scale-apimanager-install-commands.sh
+  resources:
+    requests:
+      cpu: '1'
+      memory: 500Mi
+  env:
+  - name: AWS_ACCESS_KEY_ID
```

Comment:
@calebevans 3scale-deploy repo requires these env variables set to create/delete S3 bucket
 https://github.com/3scale-qe/3scale-deploy/blob/main/README.md#variables

Reply:
Yes, but in lines 7-9 of `ci-operator/step-registry/3scale/apimanager/install/3scale-apimanager-install-commands.sh`, these values are always overridden anyway.

Reply:
okay, so meaning these variables are already set, even if we don't provide in config file?

Reply:
Right, we would only provide these env vars in this ref.yaml if we wanted to allow someone else using this ref to change these vars from the env section of their config file.

Since these vars are secrets we'd never want someone explicitly setting these in their config files so there's no need to place the vars here. These can just be considered dynamic vars that we assign from within the shell script like you are already doing.

Reply:
removed from yaml